docs/openapi_generator/pyopenapi/utility.py

```diff
@@ -135,6 +135,17 @@ def _validate_api_method_return_type(method) -> str | None:
         return "returns Optional type"
 
 
+def _validate_api_method_doesnt_return_list(method) -> str | None:
+    hints = get_type_hints(method)
+
+    if 'return' not in hints:
+        return "has no return type annotation"
+
+    return_type = hints['return']
+    if get_origin(return_type) is list:
+        return "returns a list"
```

Comment:
Should we be more descriptive and include a resolution to guide users?

Reply:
I'm adding prescriptions to all existing validators.